recipes/paraview/5.13.2/zen2/environments.yaml

```diff
@@ -0,0 +1,128 @@
+paraview-zen2-osmesa:
+  compiler:
+  - toolchain: gcc
+    spec: gcc
+  mpi:
+    spec: cray-mpich
+    gpu: Null
+  unify: true
+  views:
+    develop:
+      exclude: []
+      uenv:
+        add_compilers: true
+        prefix_paths:
+          LD_LIBRARY_PATH: [lib, lib64]
+    paraview:
+      link: run
+      exclude: []
+      uenv:
+        add_compilers: true
+        prefix_paths:
+          LD_LIBRARY_PATH: [lib, lib64]
+    paraview-python:
+      link: run
+      exclude: []
+      uenv:
+        prefix_paths:
+          LD_LIBRARY_PATH: [lib, lib64]
+
+  specs:
+
+  # Build tools
+  - git-lfs
+  - cmake
+  - ninja
+  - direnv
+  - ninja
+  - libtree
+  - llvm@18 ~gold ~cuda
+
+  # maths
+  - blaspp
+  - eigen
+  - fftw
+  - lapackpp
+  - openblas
+  - proj
+
+  # c++ helpers
+  - fmt@10
+  - boost +atomic +chrono +container +context +coroutine +date_time +filesystem +graph +json +mpi +multithreaded +program_options +regex +serialization +shared +system +test +thread
+
+  # IO and parallelism
+  - hdf5 +mpi +cxx +hl +threadsafe +shared ~java
+  - netcdf-c +mpi
+  - lz4
+  - adios2 +python +hdf5
+
+  # memory management
+  - jemalloc
+  - mimalloc
+
+  # system
+  - numactl
+
+  # profiling/testing
+  - gperftools
+  - googletest
```

Comment:
Same questions here as for the gh200 recipe.

Reply:
I do not see the other question. There were some packages added for compatibility with plugin builds, but I will remove them.

Reply:
I was referring to the comments on `gh200/environments.yaml`, i.e. https://github.com/eth-cscs/alps-uenv/pull/39#discussion_r2099712688 and https://github.com/eth-cscs/alps-uenv/pull/39#discussion_r2099714100.